examples/meta/src/multiclass/random_forest.sg

```diff
@@ -15,7 +15,8 @@ MajorityVote m_vote()
 #![create_combination_rule]
 
 #![create_instance]
-RandomForest rand_forest(features_train, labels_train, 100)
+RandomForest rand_forest(features_train, labels_train)
+rand_forest.put("num_bags", 100)
```

Comment:
Isn’t it possible to use kwargs style here?

Reply:
kwargs style works with factories only. I divided this into two lines because there are two constructors:
```
CRandomForest(CFeatures* features, CLabels* labels, int32_t num_bags=10, int32_t num_rand_feats=0);
CRandomForest(CFeatures* features, CLabels* labels, SGVector<float64_t> weights, int32_t num_bags=10, int32_t num_rand_feats=0);
```
and R has no distinction between primitives and vectors, so the wrong constructor was called.

Reply:
ah yes.
Would be good to have a factory for RF as well, but there might be some issues blocking this (although I don't know for sure)